README.rst

```diff
@@ -182,6 +182,52 @@ You can optionally pass conditions to the query::
 Note that ``order_by`` must be chosen so that the ordering is unique, otherwise there might be
 inconsistencies in the pagination (such as an instance that appears on two different pages).
 
+
+System models
+-------------
+
+`Clickhouse docs <https://clickhouse.yandex/reference_en.html#System tables>`_.
+
+System models are read only models for implementing part of the system's functionality,
+and for providing access to information about how the system is working.
+
+Usage example::
+
+    >>>> from infi.clickhouse_orm import system_models
+    >>>> print(system_models.SystemPart.get(Database()))
+
+Currently the following system models are supported:
+
+===================  ============    ===================================================
+Class                DB Table        Comments
+===================  ============    ===================================================
+SystemPart           system.parts    Gives methods to work with partitions. See below.
+===================  ============    ===================================================
+
+
+Partitions and parts
+--------------------
+
+`ClickHouse docs <https://clickhouse.yandex/reference_en.html#Manipulations with partitions and parts>`_.
+
+A partition in a table is data for a single calendar month. Table "system.parts" contains information about each part.
+
+===================  =======================    =============================================================================================
+Method               Parameters                 Comments
+===================  =======================    =============================================================================================
+get(static)          database, conditions=""    Gets database partitions, filtered by conditions
+get_active(static)   database, conditions=""    Gets only active (not detached or dropped) partitions, filtered by conditions
+detach               database, settings=None    Detaches the partition. Settings is a dict of params to pass to http request
+drop                 database, settings=None    Drops the partition. Settings is a dict of params to pass to http request
+attach               database, settings=None    Attaches already detached partition. Settings is a dict of params to pass to http request
+freeze               database, settings=None    Freezes (makes backup) of the partition. Settings is a dict of params to pass to http request
+fetch                database, settings=None    Fetches partition. Settings is a dict of params to pass to http request
+===================  =======================    =============================================================================================
+
+``Note``: system.parts stores information for all databases. To be correct,
+SystemPart model was designed to receive only given database parts.
+
+
```

Comment:
It would be nice to add a usage example, e.g. code that loads all parts, makes a backup of the oldest one and then deletes it.